pommerman/envs/v0.py

```diff
@@ -226,9 +229,21 @@ def render(self, mode='human', close=False, record_pngs_dir=None, record_json_di
         img = np.concatenate([all_img, other_imgs], 1)
 
         if self._viewer is None:
-            from gym.envs.classic_control import rendering
-            self._viewer = rendering.SimpleImageViewer()
-            self._viewer.imshow(img)
+            if mode == 'rgb_pixel':
+                self._viewer = graphics.PixelViewer(board_size=self._board_size,
```

Comment:
Does this need all of these inputs or just the img?

Reply:
I am planning on moving that logic over to the PixelViewer.